### DISCUSSION TITLE
Laravel missing API project generation command

### DISCUSSION THREAD

**Discussion by pitis** [laravel/framework#37665] [Ideas]
I searched the documentation, but couldn't find anything related to this.

How come Ruby on Rails has this, and Laravel doesn't?

Such a great framework, but I don't want webpack and all the other stuff, I just want it to create API backends. There should be a option to create one just like in rails, maybe add a `--api` parameter.

**Comment by denisdulici** [comment]
That's possible via Lumen https://github.com/laravel/lumen
